docs/guides/security/backups/create-restic-repository-shortguide/index.md

```diff
@@ -12,26 +12,30 @@ title: "Install Restic backup"
 headless: true
 show_on_rss_feed: false
 ---
+
+{{< note >}}
 [Create an Object Storage access key pair](/docs/platform/object-storage/how-to-use-object-storage/#generate-a-key-pair) if you have not done so already.
+{{< /note >}}
 
-Run the below command, replacing "your-key" with your access key, "your-secret" with your key's secret and "your-bucket-name" with the name of your bucket:
+1. Run the below command, replacing `your-key` with your access key, `your-secret` with your key's secret, and `your-bucket-name` with the name of your bucket:
 
-    AWS_ACCESS_KEY_ID=your-key AWS_SECRET_ACCESS_KEY=your-secret restic -r s3:us-east-1.linodeobjects.com/your-bucket-name init
+        AWS_ACCESS_KEY_ID=your-key AWS_SECRET_ACCESS_KEY=your-secret restic -r s3:us-east-1.linodeobjects.com/your-bucket-name init
+
+    {{< note >}}
```

Comment:
Re-worded this note for a very slight subtlety. In the future, we may have more than one object storage cluster in a data center. For example, Newark, NJ might also have a `us-east-2` cluster. So, the new wording tries to accommodate that